.pre-commit-config.yaml

```
@@ -102,6 +102,22 @@ repos:
       - id: yamllint
         files: ^tmt/schemas/.*\.yaml
 
+  - repo: https://github.com/ansible-community/ansible-lint.git
+    rev: v24.2.0
```

Update got released this week

```suggestion
    rev: v24.2.1
```

---

Updated. Thanks for review!